client/src/components/ui/CustomMap.js

```diff
@@ -11,6 +11,9 @@ import PropTypes from 'prop-types';
 
 const CustomMap = function(props) {
   const [selectedLandmark, setSelectedLandmark] = useState(null);
+  if (Object.keys(props.markers).length === 0) {
```

Comment:
For booleans that are 0 vs 1, you can just use 

`if (!Object.keys(props.markers).length)`